week-1-practise/level-2/1-tax.js

```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(price) {
+  return price * (20 / 100);
```

Comment:
Excellent! I like that it is very clear that it is adding 20%.

There is a way to write this without both a multiplication and a division. It can be done with only 1 operation. Can you comment to show me how?